lib/zendesk/comment.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Zendesk.Comment do
+  alias Zendesk.Ticket
+  alias Zendesk.Client.{Operation, Result}
+  alias __MODULE__
+
+  defstruct [
+    :attachments,
+    :audit_id,
+    :author_id,
+    :body,
+    :created_at,
+    :html_body,
+    :id,
+    :metadata,
+    :plain_body,
+    :public,
+    :type,
+    :via
+  ]
+
+  @type t :: %__MODULE__{}
+
+  @doc """
+  Get a list of `Zendesk.Macro`s.
+
+  If passed to `Zendesk.request!/1` it will return only the first results from the list.
+  You can create a `Stream` to paginate over all results by calling `Zendesk.stream!/1`.
+
+  For instance:
+
+       # get 400 macros
+       Zendesk.Macro.list()
+       |> Zendesk.stream!()
+       |> Stream.take(400)
+       |> Enum.to_list()
+       |> IO.inspect()
+  """
+  @spec list_for(Ticket.t()) :: Operation.t()
+  def list_for(%Ticket{id: id}) do
+    %Operation{path: "tickets/#{id}/comments.json", parser: &parse_list/1}
+    |> Operation.with_page_size(1)
+  end
+
+  @doc false
+  def parse(%Result{parsed: %{macro: macro}}), do: {:ok, struct(Macro, macro)}
```

Comment:
```suggestion
def parse(%Result{parsed: %{comment: comment}}), do: {:ok, struct(Comment, comment)}
```

Reply:
This function can actually just be removed - AFAIK there's no way to get just one comment.

Reply:
Done in 594d345